pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -881,3 +885,29 @@ func getIntOrPercentValue(intOrStr *intstr.IntOrString) (int, bool, error) {
 	}
 	return 0, false, fmt.Errorf("invalid type: neither int nor percentage")
 }
+
+func (r *ReconcileMachineHealthCheck) isAlertOldEmr(emr *unstructured.Unstructured) bool {
```

Comment:
IIRC the "is" prefix is quite common for bool type methods/variables.
Not to say it's wrong but I haven't encountered the same usage for "should/was".
There is a very good chance that I'm completely wrong here... and I'm completely out of sync with current conventions :smile: 
Can you point me to anything to support this usage, or do you think that in this case using "should" is just common sense ?

Reply:
I wasn't relying on any formal source, just my personal view.
for example, `isAlertedSent` doesn't sound right to me, but `wasAlertSent` sounds better. 

Apple's coding guidelines allow `should` etc.:
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-BCIGIJJF

But since this is just a convention I'm sure there will be other sources saying this is discouraged :)

the bottom line - do whatever you think fit, as long as the repo maintainers allow it.